### ISSUE TITLE
Support for getting the user's default view

### ISSUE THREAD

**Issue by jyothimandava** [tableau/server-client-python#702]
Hi ,
After retrieving specific user's workbooks, is there support for retrieving the user's default view of those workbook(s).  It will be great to share an example if it is supported.  If it's not is there please share if there is any plan for adding this support? 
Thanks

**Comment by bcantoni** [comment]
I don't believe the backend REST API supports this yet, so I'll label this one as a backend enhancement request.

**Comment by jacalata** [comment]
Default view ID is now an attribute when you get a workbook - I'm not sure if that will reflect any user-specific defaults though. 

There are also now a couple methods around default views, such as https://help.tableau.com/current/api/rest_api/en-us/REST/rest_api_ref_workbooks_and_views.htm#list_users_with_custom_view_as_default

**Comment by jacalata** [comment]
Update on this after re-checking: the ask has actually become addressable at the REST layer since this issue was filed, but TSC hasn't caught up.

The REST API added two Custom View default endpoints in version 3.18 (Tableau Cloud December 2022 / Server 2023.1):

- `GET /api/{api-version}/sites/{site-luid}/customviews/{custom-view-luid}/defaultFor` — list users who have a specified custom view set as their default
- `PUT /api/{api-version}/sites/{site-luid}/customviews/{custom-view-luid}/setAsDefault` — set a custom view as the default for a list of users

These are documented at https://help.tableau.com/current/api/rest_api/en-us/REST/rest_api_ref_workbooks_and_views.htm

Neither is exposed on `server.custom_views` in TSC today (verified against current `custom_views_endpoint.py` on `development`).

The API shape differs from the original ask ("get user X's default view for workbook Y") — it's custom-view-centric rather than user-centric — but it does cover the underlying use case of programmatically managing per-user default views. Reclassifying this as a TSC implementation gap rather than a server-side pending item; keeping open.
